shenyu-kubernetes-controller/src/main/java/org/apache/shenyu/k8s/parser/DivideIngressParser.java

```diff
@@ -0,0 +1,357 @@
+                            .loged(false)
+                            .enabled(true).build();
+
+                    res.add(Pair.of(selectorData, ruleData));
+                }
+            }
+        }
+        return res;
+    }
+
+    private String parsePort(final V1IngressServiceBackend service) {
+        if (Objects.nonNull(service.getPort())) {
+            if (service.getPort().getNumber() != null && service.getPort().getNumber() > 0) {
+                return String.valueOf(service.getPort().getNumber());
+            } else if (service.getPort().getName() != null && !"".equals(service.getPort().getName().trim())) {
+                return service.getPort().getName().trim();
+            }
+        }
+        return null;
+    }
+
+    private List<DivideUpstream> parseUpstream(final V1IngressBackend backend, final String namespace) {
+        List<DivideUpstream> upstreamList = new ArrayList<>();
+        if (Objects.nonNull(backend) && Objects.nonNull(backend.getService()) && Objects.nonNull(backend.getService().getName())) {
+            String serviceName = backend.getService().getName();
+            // shenyu routes directly to the container
+            V1Endpoints v1Endpoints = endpointsLister.namespace(namespace).get(serviceName);
+            List<V1EndpointSubset> subsets = v1Endpoints.getSubsets();
+            if (Objects.isNull(subsets) || CollectionUtils.isEmpty(subsets)) {
+                LOG.info("Endpoints {} do not have subsets", serviceName);
+            } else {
+                for (V1EndpointSubset subset : subsets) {
+                    List<V1EndpointAddress> addresses = subset.getAddresses();
+                    if (Objects.isNull(addresses) || addresses.isEmpty()) {
+                        continue;
+                    }
+                    for (V1EndpointAddress address : addresses) {
+                        String upstreamIp = address.getIp();
+                        String defaultPort = parsePort(backend.getService());
+                        if (Objects.nonNull(defaultPort)) {
+                            DivideUpstream upstream = new DivideUpstream();
+                            upstream.setUpstreamUrl(upstreamIp + ":" + defaultPort);
+                            upstream.setWeight(100);
+                            // TODO support config protocol in annotation
+                            upstream.setProtocol("http://");
+                            upstream.setWarmup(0);
+                            upstream.setStatus(true);
+                            upstream.setUpstreamHost("");
+                            upstreamList.add(upstream);
+                        }
+                    }
+                }
+            }
+        }
+        return upstreamList;
+    }
+
+    private Pair<SelectorData, RuleData> getDefaultRouteConfig(final List<DivideUpstream> divideUpstream, final Map<String, String> annotations) {
+        String id = UUIDUtils.getInstance().generateShortUuid();
```

Comment:
For the globally unique default route, I think we can set a fixed value when setting the ID, at least the ID values of multiple Controllers must be consistent. We still lack a cluster state consistency scheme (such as the election of the master), but using random uuid is obviously not good. So I suggest temporarily setting the id here to a fixed value such as "1", "2", "3", which can be set as a constant and placed in `IngressConstans`.